packages/lesswrong/lib/notificationTypes.tsx

```diff
@@ -363,6 +373,31 @@ export const NewGroupOrganizerNotification = registerNotificationType({
   }
 })
 
+export const NewCommentOnDraftNotification = registerNotificationType({
```

Comment:
Creates a new type of notification for when a user receives a comment on a draft post